.circleci/config.yml

```diff
@@ -18,12 +18,16 @@ jobs:
           name: Ensure markdown tables of contents are up to date
           command: ./.circleci/doctoc-check.sh
 
-  ingestion-edge:
+  ingestion-edge: &ingestion-edge
+    working_directory: /root/project/ingestion-edge
     docker:
     - image: python:3.7
     steps:
-    - checkout
-    - run: &skip_unmodified
+    - &checkout
+      checkout:
+        path: /root/project
```

Comment:
This is very nice to clean up the invocations. I did not know setting a checkout path was possible.